src/Components/Modal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react' 
+import './components.css';
+import CloseButton from './CloseButton';
+import CloseParagraph from './CloseParagraph';
+
+/**
+ * This is a component that return a modal interface
+ */
+
+function Modal({setModalOpen}){
+    const modalBody = `Lorem ipsum dolor sit amet,consectetur adipiscing elit,
+sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+ Felis imperdiet proin fermentum leo vel orci porta non pulvinar.`;
+    const modalTitle = `Esto es un modal`;
+
+    return (
+    <article className="modalBackground">
+        <div className="modalContainer">
+            <div className="titleCloseBtn">
+                <div className="closeButton" onClick={() => setModalOpen(false)}>
+                    <CloseButton />
```

Comment:
One of the features we require is this button to be configurable but it's hardcoded.  Can you think of a way to achieve that? Also, notice that you are handling the click in a wrapper and not in the close button element.

Reply:
In my last commit I remove the onClick logic and I only send a property with the useState to do the logic in the CloseButton component.